shaders/abuffer/resolveabuffer.vert

```diff
@@ -29,5 +29,5 @@ out vec2 texCoord;
 
 void main() {
     gl_Position = position;
-    texCoord = 0.5 + position.xy / 2.0;
+    texCoord = 0.5 + position.xy * 0.5;
```

Comment:
It is true for a CPU's compiler. The GPU's compiler seems to handle this as a conventional division. There is performance gains doing the multiplication.